rllib/core/learner/learner.py

```diff
@@ -1045,36 +1041,13 @@ def additional_update_for_module(
         Returns:
             A dictionary of results from the update
         """
-        if hps is not None:
-            deprecation_warning(
-                old="Learner.additional_update_for_module(.., hps=..)",
-                help="Deprecated argument. Use `config` (AlgorithmConfig) instead.",
-                error=True,
-            )
-
-        # Only cover the optimizer mapped to this particular module.
-        for optimizer_name, optimizer in self.get_optimizers_for_module(module_id):
-            # Only update this optimizer's lr, if a scheduler has been registered
-            # along with it.
-            if optimizer in self._optimizer_lr_schedules:
-                new_lr = self._optimizer_lr_schedules[optimizer].update(
-                    timestep=timestep
-                )
-                self._set_optimizer_lr(optimizer, lr=new_lr)
-
-                # Make sure our returned results differentiate by optimizer name
-                # (if not the default name).
-                stats_name = LEARNER_RESULTS_CURR_LR_KEY
-                if optimizer_name != DEFAULT_OPTIMIZER:
-                    stats_name += "_" + optimizer_name
-                self.metrics.log_value(
-                    key=(module_id, stats_name), value=new_lr, window=1
-                )
+        pass
 
     def update_from_batch(
         self,
         batch: MultiAgentBatch,
         *,
+        timesteps=None,  # TODO
```

Comment:
this is still a TODO for this PR:
* We should pass in the current global timesteps (env, agents and modules) into all `update` calls
* instead, we skip the `additional_update` call and replace it with a user-overridable before_update and after_update sub-method, called automatically by `Learner.update_from_batch|episodes`.
* In the future, we might simplify by deprecating `update_from_batch` and fully replace it with `update`, which will take only episodes. ??

Reply:
As we fully focus on episodes this makes sense in my opinion.